### ISSUE TITLE
If I call browser.actions() multiple times, modifier keys don't work frequently.

### ISSUE THREAD

**Issue by crocket** [angular/protractor#1355]
``` javascript
// The first ActionSequence instance work correctly.
browser.actions().sendKeys(protractor.Key.CONTROL, 's').perform();

// Ctrl is not pressed in the second ActionSequence instance.
browser.actions().sendKeys(protractor.Key.CONTROL, protractor.Key.SHIFT, 'm').perform();
```

It seems multiple instances of ActionSequence conflict with each other.

If I invoke browser.actions() only once and reuse it, there is another issue.

``` javascript
var actions = browser.actions();

// This works correctly
actions.sendKeys(protractor.Key.CONTROL, 's').perform();

// This works, but 's' is pressed into an HTML editor. 's' is probably from the last action sequence.
actions.sendKeys(protractor.Key.CONTROL, protractor.Key.SHIFT, 'm').perform();

// In this action sequence, CONTROL is not pressed, thus I end up seeing 'o' in the HTML editor.
actions.sendKeys(protractor.Key.CONTROL, 'o').perform();
```

Is it a bug in protractor or selenium webdriver?
